PRIVACY.txt

```diff
@@ -14,7 +14,7 @@ We receive different types of information from our Products. We use this data to
 
 We may also receive your personal contact information contained in correspondence via our web sites and communications channels. This enables us to offer user support, beta testing communities, surveys, competitions and similar services.
 
-We will collect your email address, you subscriptions and what do listen on our Products to enable services such as personalization, prediction, synchronization and backup. We may use data provided by you to develop and improve our Products.
+We will collect your email address, you subscriptions, and what you listen using our Products to enable services such as personalization, prediction, synchronization and backup. We may use data provided by you to develop and improve our Products.
```

Comment:
Are you thinking something like ", your subscriptions, and all of the episodes you listen using our Products"?